orca-core/src/main/java/com/netflix/spinnaker/orca/pipeline/tasks/EvaluateVariablesTask.java

```diff
@@ -36,6 +36,11 @@
   public EvaluateVariablesTask() {
   }
 
+  /**
+   * Copies previously evaluated expressions to the outputs map for consumption by subsequent stages.
+   * The variables aren't evaluated here because would've been evaluated already by a call to
+   * e.g. ExpressionAware.Stage#withMergedContext
+   */
```

Comment:
maybe better put on the class level docs?

Reply:
i was going back the forth on this :)... happy to move it